### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by younesStrittmatter**
added runner components

### CODE REVIEW

**Review by musslick — CHANGES_REQUESTED** [review]
Looks great so far. What about adding the synthetic runners, e.g.,

autora[experiment-runner-synthetic-abstract-equation]?

**Comment by younesStrittmatter** [comment]
You are right. Forgot that one and added it now.

File: README.md [hunk 1]
```diff
@@ -30,5 +30,9 @@ Experimentalist Components:
 - [autora[experimentalist-uncertainty]](https://github.com/AutoResearch/autora-experimentalist-uncertainty): Automated experimental design based on model uncertainty.
 
 Experiment Runner Components:
+- [autora[experiment-runner-firebase-prolific]](https://github.com/AutoResearch/autora-experiment-runner-firebase-prolific): Automated data collection through serving an experiment on [Firebase](https://firebase.google.com/) and recruiting participants with [Prolific](https://www.prolific.com/).
+- [autora[experiment-runner-experimentation-manager-firebase]](https://github.com/AutoResearch/autora-experiment-runner-experimentation-manager-firebase): Automated data collection through serving an experiment on Firebase.
+- [autora[experiment-runner-recruitment-manager-prolific]](https://github.com/AutoResearch/autora-experiment-runner-recruitment-manager-prolific): Automated participant recruitment with Prolific.
+- [autora[experiment-runner-synthetic-abstract-equation]](https://github.com/AutoResearch/autora-synthetic-abstract-equation): Automated data generation based on mathematical equations.
```
Comment (musslick) [inline README.md:36]:
```suggestion
- [autora[experiment-runner-synthetic-abstract-equation]](https://github.com/AutoResearch/autora-synthetic-abstract-equation): Automated data generation based on mathematical equations that can be sampled.
```

**Review by musslick — APPROVED** [review]
Looks great! Just made a minor suggestion

**Pull request merged**